subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/dependencies/DefaultDependencyHandler.java

```diff
@@ -266,6 +266,11 @@ public Dependency gradleApi() {
         return dependencyFactory.createDependency(DependencyFactory.ClassPathNotation.GRADLE_API);
```

Comment:
Should we qualify the name of this constant now? Like, `CURRENT_GRADLE_API`? Or `BUNDLED_GRALDE_API`?

Reply:
I wouldn't do this now. I would like to get rid of this notation and always use a real dependency.